widget/textgrid.go

```diff
@@ -161,6 +171,7 @@ func (t *TextGrid) Row(row int) TextGridRow {
 
 // RowText returns a string representation of the content at the row specified.
 // If the index is out of bounds it returns an empty string.
+// Tab characters do not have padding spaces removed.
```

Comment:
It seems like we can (and should) fix this rather than documenting the exception.

Reply:
Done